src/Device/Port/SerialPort.cpp

```diff
@@ -61,8 +62,11 @@ SerialPort::Open(const TCHAR *path, unsigned _baud_rate)
 
   DCB PortDCB;
 
+  StaticString<0x20> port;
+  port.Format(_T("\\\\.\\%s"), path);
+
   // Open the serial port.
-  hPort = CreateFile(path,
+  hPort = CreateFile(port.c_str(),
```

Comment:
This violates this class's API, because it opens a different path than the one specified by the caller.

Reply:
Maybe, but this is a common way on Windows to select a serial port. I don't know any program, where I select the complete portname, every time the choice is 'COM3' or 'COM25' and so on. If you want to have the complete portname in the setting file (profile), you have to add this '\\\\.\\' after the selection in the port list, but this make not so much sense for me and makes the profile file more incomprehensible from my point of view!

Reply:
I hoped to solve this problem on the root - without any API limitations. Maybe this is wrong. I will rebuild this part!

Reply:
Building the device name moved to Device/Port/ConfiguredPort.cpp to fullfill the API